<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="gen1Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Generacion 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen5Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Generacion 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen2Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Generacion 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen6Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Generacion 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen3Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Generacion 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen7Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Generacion 7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen4Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Generacion 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen8Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Generacion 8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pok√©dex</string>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>801</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../EXTRAS/fondo.jpg</pixmap>
    </property>
   </widget>
   <zorder>img</zorder>
   <zorder>gen1Button</zorder>
   <zorder>gen5Button</zorder>
   <zorder>gen2Button</zorder>
   <zorder>gen6Button</zorder>
   <zorder>gen3Button</zorder>
   <zorder>gen7Button</zorder>
   <zorder>gen4Button</zorder>
   <zorder>gen8Button</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
